src/pages/SingleBreedPage/singleBreed.scss

```diff
@@ -0,0 +1,26 @@
+.single-breed {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  min-height: 90vh;
+  padding: 0 25px;
+  text-transform: capitalize;
+
+  &__link {
+    text-decoration: none;
+    display: inline-block;
+    transition: all 0.3s ease-in-out;
+    color: #035867;
```

Comment:
use scss color variables if possible, I met this color #035867 a couple of times in your stylesheet